diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -2637,7 +2637,7 @@ export class ContentGame extends React.Component {
                                                                 key={"r"}
                                                                 pickEvent={true}
                                                                 title={
-                                                                    "dismiss"
+                                                                    String.fromCharCode(0x2716)
```

Comment:
Why is this "dismiss" a different character from the others?

Also, would it be possible for you to just use the character directly in a string literal? That would be easier to read because I definitely don't know that hex code from memory!